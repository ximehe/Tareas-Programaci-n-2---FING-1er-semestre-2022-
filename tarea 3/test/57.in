 # implementar menores
menores 20.0
insertarEnAbb (20,20.00)
menores 20.0
menores 21.0
insertarEnAbb (15,25.0)
menores 21.0
menores 25.0
menores 26.0
insertarEnAbb (10,10.0)
menores 20.0
insertarEnAbb (40,11.0)
menores 11.0
menores 12.0
insertarEnAbb (50,5.0)
menores 10.0
menores 11.0
insertarEnAbb (30,15.0)
insertarEnAbb (25,6.0)
insertarEnAbb (35,10.0)
menores 10.0
menores 11.0
menores 6.0
insertarEnAbb (32,5.0)
menores 6.0
menores 12.0
insertarEnAbb (27,4.0)
menores 5.0
menores 12.0
menores 11.0
Fin