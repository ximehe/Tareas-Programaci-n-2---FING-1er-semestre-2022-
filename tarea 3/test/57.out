1>#  implementar menores.
2>
3>Se insertó (20,20.00) en el abb.
4>
5>
(20,20.00)
6>Se insertó (15,25.00) en el abb.
7>
(20,20.00)
8>
(20,20.00)
9>
(20,20.00)
-(15,25.00)
10>Se insertó (10,10.00) en el abb.
11>
(10,10.00)
12>Se insertó (40,11.00) en el abb.
13>
(10,10.00)
14>
-(40,11.00)
(10,10.00)
15>Se insertó (50,5.00) en el abb.
16>
(50,5.00)
17>
-(50,5.00)
(10,10.00)
18>Se insertó (30,15.00) en el abb.
19>Se insertó (25,6.00) en el abb.
20>Se insertó (35,10.00) en el abb.
21>
-(50,5.00)
(25,6.00)
22>
--(50,5.00)
-(35,10.00)
--(25,6.00)
(10,10.00)
23>
(50,5.00)
24>Se insertó (32,5.00) en el abb.
25>
-(50,5.00)
(32,5.00)
26>
--(50,5.00)
-(40,11.00)
---(35,10.00)
----(32,5.00)
--(25,6.00)
(10,10.00)
27>Se insertó (27,4.00) en el abb.
28>
(27,4.00)
29>
--(50,5.00)
-(40,11.00)
---(35,10.00)
----(32,5.00)
--(27,4.00)
---(25,6.00)
(10,10.00)
30>
--(50,5.00)
-(35,10.00)
---(32,5.00)
--(27,4.00)
---(25,6.00)
(10,10.00)
31>Fin.
